quickstart/101-front-door-classic/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "location" {
+  type    = string
+  default = "westus2"
+}
+
+variable "resource_group_name" {
+  type    = string
+  default = "FrontDoor"
+}
+
+variable "backend_address" {
+  type = string
+}
+
+resource "random_id" "front_door_name" {
```

Comment:
This resource should not be in the variables.tf file. I see why you put it here since there really is no perfect place for it but I would suggest putting it in the resource-group.tf file.

Reply:
Updated.